src/NzbDrone.Core/Tv/AddSeriesService.cs

```diff
@@ -78,13 +78,13 @@ public List<Series> AddSeries(List<Series> newSeries, bool ignoreErrors = false)
                     series.Added = added;
                     if (existingSeriesTvdbIds.Any(f => f == series.TvdbId))
                     {
-                        _logger.Debug("TVDB ID {0} was not added due to validation failure: Series already exists in database", s.TvdbId);
+                        _logger.Debug("TVDB ID {0} was not added due to validation failure: Series {1} already exists in database", s.TvdbId, s);
                         continue;
                     }
 
                     if (seriesToAdd.Any(f => f.TvdbId == series.TvdbId))
                     {
-                        _logger.Debug("TVDB ID {0} was not added due to validation failure: Series already exists on list", s.TvdbId);
+                        _logger.Debug("TVDB ID {0} was not added due to validation failure: Series {1} already exists on list", s.TvdbId, s);
```

Comment:
I'm not sure how much value this message actually adds. It's not being added because it's on another list that was processed first, but the series will still be added, but at worst it has different monitoring.

Reply:
Should we demote to trace?

Reply:
Yeah and tweaking the messaging.

```suggestion
                        _logger.Trace("TVDB ID {0} was already added from another import list, not adding series {1} again", s.TvdbId, s);
```